### PR TITLE
added Marital Status Valueset

### PR DESCRIPTION
**Pull request by saulakravitz**
still needs to be removed from VRDR....but it is added here.
https://jira.hl7.org/browse/FHIR-43416

### CODE REVIEW

File: input/pagecontent/content-transitions.md
Comment (ptran265) [inline input/pagecontent/content-transitions.md:1]: I think the link for mother relationship status is mistaken

**Review by ptran265 — CHANGES_REQUESTED** [review]

File: input/fsh/valuesets/ValueSetMaritalStatusVitalRecords.fsh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Title: "Marital Status Value Set"
+Description: "The set of codes used to indicate the marital status of the decedent
+
+Mapping to IJE codes [here](ConceptMap-MaritalStatusCM.html)."
+* $v3-MaritalStatus#D "Divorced"
+* $v3-MaritalStatus#L "Legally Separated"
+* $v3-MaritalStatus#M "Married"
+* $v3-MaritalStatus#S "Never Married"
+* $v3-MaritalStatus#W "Widowed"
+* $v3-MaritalStatus#T "Domestic Partnership" // not currently supported by NCHS
```
Comment (rpassas) [inline input/fsh/valuesets/ValueSetMaritalStatusVitalRecords.fsh:12]: Should it be included in the VS if not supported?

**Review by rpassas — APPROVED** [review]

File: input/pagecontent/content-transitions.md
Comment (rpassas) [reply input/pagecontent/content-transitions.md:1]: should be http://hl7.org/fhir/us/vrdr/STU2.2/ValueSet-vrdr-marital-status-vs.html

File: input/fsh/valuesets/ValueSetMaritalStatusVitalRecords.fsh (hunk 1 above)
Comment (saulakravitz) [reply input/fsh/valuesets/ValueSetMaritalStatusVitalRecords.fsh:12]: Probably more confusing not to....I'll comment it out...

File: input/pagecontent/content-transitions.md
Comment (rpassas) [reply input/pagecontent/content-transitions.md:1]: I think it stays as http://hl7.org/fhir/us/vr-common-library/STU1.1/ValueSet-ValueSet-mother-relationship-vr.html, right?

File: input/fsh/valuesets/ValueSetMaritalStatusVitalRecords.fsh (hunk 1 above)
Comment (rpassas) [reply input/fsh/valuesets/ValueSetMaritalStatusVitalRecords.fsh]: I'll defer to your better judgement

**Review by ptran265 — APPROVED** [review]

**Pull request merged**